### PR TITLE
feat(route): add 竹白blog

### PR DESCRIPTION
**Pull request by naixy28**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/zhubai/via
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [x] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/zhubai/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            Referer: `https://${id}.zhubai.love/`,
+        },
+    });
+    const data = response.data.data;
+    const { name, description } = data[0].publication;
+
+    ctx.state.data = {
+        title: String(name),
+        link: `https://${id}.zhubai.love/`,
+        description: String(description),
```
Comment (TonyRL) [inline lib/v2/zhubai/index.js:20]:
```suggestion
        title: name,
        link: `https://${id}.zhubai.love/`,
        description,
```

File: lib/v2/zhubai/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        item: data.map((item) => ({
+            title: item.title,
+            pubDate: new Date(item.created_at).toUTCString(),
```
Comment (TonyRL) [inline lib/v2/zhubai/index.js:23]: Please use parseDate()
docs: https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei-parse-date

File: docs/blog.md [hunk 3]
```diff
@@ -262,3 +262,9 @@ pageClass: routes
 ### 分类
 
 <Route author="XinRoom" example="/ddosi/category/黑客工具" path="/ddosi/category/:category?"/>
+
+## 竹白
+
+### 文章
+
+<Route author="naixy28" example="/blogs/zhubai/via" path="/blogs/zhubai/:id"/>
```
Comment (TonyRL) [inline docs/blog.md:270]: Please explain what `id` is in `paramsDesc`

File: lib/v2/zhubai/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const limit = 20;
```
Comment (TonyRL) [inline lib/v2/zhubai/index.js:5]: Consider allowing user changing this number using `ctx.query.limit`
https://docs.rsshub.app/parameter.html#tiao-shu-xian-zhi

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/blog.md [hunk 5]
```diff
@@ -262,3 +262,10 @@ pageClass: routes
 ### 分类
 
 <Route author="XinRoom" example="/ddosi/category/黑客工具" path="/ddosi/category/:category?"/>
+
+## 竹白
+
+### 文章
+
+<Route author="naixy28" example="/blogs/zhubai/via" path="/blogs/zhubai/:id"  :paramsDesc="['id为竹白主页url中的三级域名，如 via.zhubai.love 的id为 via']"/>
+> 在路由末尾处加上 `?limit=限制获取数目` 来限制获取条目数量，默认值为`20`.
```
Comment (TonyRL) [inline docs/blog.md:271]:
```suggestion
<Route author="naixy28" example="/blogs/zhubai/via" path="/blogs/zhubai/:id"  :paramsDesc="['`id` 为竹白主页 url 中的三级域名，如 via.zhubai.love 的 `id` 为 `via`']">

::: tip 提示
在路由末尾处加上 `?limit=限制获取数目` 来限制获取条目数量，默认值为`20`
::: 

</Route>
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/zhubai/index.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const limit = parseInt(ctx.query.limit) || 20;
```
Comment (TonyRL) [inline lib/v2/zhubai/index.js:6]:
```suggestion
    const limit = parseInt(ctx.query.limit) ?? 20;
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/zhubai/index.js (hunk 6 above)
Comment (naixy28) [reply lib/v2/zhubai/index.js:6]: Hi Tony, seems using `??` won't pass Deep Scan. For`parseInt()` will return either `number` or `NaN`, which would never return right-hand value when they are used as left-hand value.
```
NaN ?? 1 = NaN
```
Not sure if you have any further consideration.

Comment (TonyRL) [reply lib/v2/zhubai/index.js:6]: `const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 20`

File: docs/blog.md [hunk 7]
```diff
@@ -262,3 +262,13 @@ pageClass: routes
 ### 分类
 
 <Route author="XinRoom" example="/ddosi/category/黑客工具" path="/ddosi/category/:category?"/>
+
+## 竹白
+
+### 文章
+
+<Route author="naixy28" example="/blogs/zhubai/via" path="/blogs/zhubai/:id"  :paramsDesc="['`id` 为竹白主页 url 中的三级域名，如 via.zhubai.love 的 `id` 为 `via`']">
```
Comment (TonyRL) [inline docs/blog.md:270]:
```suggestion
<Route author="naixy28" example="/zhubai/via" path="/zhubai/:id"  :paramsDesc="['`id` 为竹白主页 url 中的三级域名，如 via.zhubai.love 的 `id` 为 `via`']">
```

**Review by TonyRL — APPROVED** [review]

**Pull request merged**